openid-4-verifiable-presentations-1_0.md

```diff
@@ -2001,68 +2028,263 @@ Note: The `nonce` and `aud` are set to the `nonce` of the request and the Client
+* Reference: (#vp_token_request) of this specification
+
+### presentation_definition_uri
+
+* Name: presentation_definition_uri
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### client_metadata
+
+* Name: client_metadata
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### request_uri_method
+
+* Name: request_uri_method
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### wallet_nonce
+
+* Name: wallet_nonce
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#request_uri_method_post) of this specification
+
+### vp_token
+
+* Name: vp_token
+* Parameter Usage Location: authorization response
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#response-parameters) of this specification
+
+### presentation_submission
+
+* Name: presentation_submission
+* Parameter Usage Location: authorization response
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#response-parameters) of this specification
+
+### response_uri
+
+* Name: response_uri
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#response_mode_post) of this specification
+
+### expected_origins
 
-The Internet media type for a Verifier Attestation JWT is `application/verifier-attestation+jwt`.
+* Name: expected_origins
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#browser_api_request) of this specification
+
+### id_token_type
+
+* Name: id_token_type
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#siop_request) of this specification
+
+## OAuth Extensions Error Registry
+
+This specification registers the following errors
+in the IANA "OAuth Extensions Error" registry [@IANA.OAuth.Parameters]
+established by [@!RFC6749].
 
-Type name: : `application`
+### vp_formats_not_supported
 
-Subtype name: : `verifier-attestation+jwt`
+* Name: vp_formats_not_supported
+* Usage Location: authorization endpoint
```

Comment:
for the same reasons given in vp_token registration, this can be token endpoint as well.

same for other errors in this section